### PR TITLE
scheme progress - bugfix ~s, modernise colours, use xhtml <br/>

### PR DESCRIPTION
**Pull request by christopherlam**
- Longstanding 'bug' in aging.scm - only matters if there are unpaid invoices dated earlier than 1970. It's not worth fixing in maint is it?
- `<br>` to `<br/>`
- `~s` to `~a` in (format)
- modernize colours to new red/blue/green from http://clrs.cc/ - from http://i.imgur.com/81U9huO.png to http://i.imgur.com/RocVPUz.png :)

### CODE REVIEW

**Comment by christopherlam** [comment]
I've also added some changes to borrowed -- but I can't understand CMakeLists.txt.
- move jqplot to borrowed, and upgrade to latest 1.0.9 - most plugins are unchanged, some new ones
- add guile-json

- jqplot would still be copied to ${CMAKE_INSTALL_DATADIR}/gnucash/jqplot wouldn't it?
- guile-json - we must be able to (use-modules (json builder)) from any scheme file, how to add this external module into the build dir?

File: gnucash/report/locale-specific/us/taxtxf.scm [hunk 1]
```diff
@@ -3452,29 +3449,29 @@
                         (gnc:html-markup-p
                          (gnc:html-markup/format
                           (string-append
-                             "Selected Report Options:<BR>"
+                             "Selected Report Options:<BR />"
```
Comment (gjanssens) [inline gnucash/report/locale-specific/us/taxtxf.scm]: While you're changing the html BR notes, can you lowercase them at the same time ?

Comment (christopherlam) [reply gnucash/report/locale-specific/us/taxtxf.scm]: Ok. There are lots of <br> elsewhere which I haven't touched because they're translatable.

**Comment by christopherlam** [comment]
I think these are clean and I'd hope the colours are welcome!

**Comment by gjanssens** [comment]
Thanks!
I agree it's not worth backporting the date issue to maint.

**Comment by christopherlam** [comment]
Ok. Perhaps I'll rebase any further scheme work onto mas ter instead, unless I come across any glaring bugs.

**Comment by gjanssens** [comment]
Yes, that's preferable.

**Pull request merged**